design/one-pager-stack-cli.md

```diff
@@ -0,0 +1,79 @@
+# Crossplane Stacks CLI
+* Owner: Daniel Suskin (@suskin)
+* Reviewers: Crossplane Maintainers
+* Status: Draft
+
+## Introduction
+*Why does this document exist?*
+
+We're working on the first version of a user experience for working with crossplane stacks. This document answers the questions of:
+* What are our goals for this version?
+* How do we measure success?
+* What do we think the experience will look like?
+
+## OKRs
+*What are our goals for this version? How do we measure success?*
+
+Objectives:
+* Provide a complete and simple experience for creating stacks from scratch (for a single pattern of creating stacks)
+* The stacks experience can be updated independently of the Crossplane releases
+
+Key Results:
+* Creating a Hello World stack from scratch takes 5 minutes, using the recommended approach
+* Creating and publishing a Hello World stack from scratch takes 10 minutes, using the recommended approach
+* All documentation linking from Crossplane 0.3 to the stacks experience will not need to be changed if we update the stacks experience
```

Comment:
Does this suggest that the documentation for the two projects are distinct? At first thought, I think they shouldn't be too isolated from each other.  Different pages in the same read-through.

Reply:
It doesn't matter whether they're distinct or not; the goal was that the toolchain releases would be decoupled from Crossplane releases. However, I **was** imagining that the documentation would be distinct when I wrote this, especially since the code lives in a separate repository. What are you imagining?

Reply:
I'm imagining Crossplane CLI instructions being included with the standard Crossplane usage and example docs, perhaps interwoven as needed.  

"Let's deploy Wordpress on GCP. First let's `kubectl crossplane stack install crossplaneio/stack-gcp.. More on the crossplane kubectl plugin on page X"

Reply:
I would like to embed the CLI instructions in with the Crossplane docs for convenience, but a **full** embedding would mean that the instructions would be unchanged for the entirety of the 0.3 release. I'd like to avoid that so that we can leave our options open once we start getting feedback from users. Once the CLI is a bit more stable, or if we have short release cycles for Crossplane, I would be more okay with it.

It would probably be okay to have some less-than-full embedding that is also more than zero embedding. I am not entirely sure what the Best Amount would be right now. Listing the installation instructions for the plugins seems like it should be fine . . . not sure about the usage.